package.json

```diff
@@ -37,7 +37,7 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "./node_modules/.bin/mocha --timeout 5000 -R spec",
+    "test": "./node_modules/.bin/mocha --timeout 9999 -R spec",
```

Comment:
Do we know why the tests are failing on a smaller timeout? Extensively increasing the timeout to this big is not a good solution.